### PR TITLE
Changed readme a bit to be useful

### PR DESCRIPTION
**Pull request by matanshukry**
Resolve issues #1

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -68,3 +86,50 @@ func getGIF(text string) string {
...
+
+	defer resp.Body.Close()
+
+	log.Println("response status: ", resp.Status)
+	log.Println("response Headers:", resp.Header)
+	body, _ := ioutil.ReadAll(resp.Body)
+	log.Println("response Body:", string(body))
+
+	var jsonResponse indicoResponse
+	// var jsonResponse map[string]interface{}
```
Comment (bennyz) [inline main.go]: Please remove

File: main.go [hunk 2]
```diff
@@ -68,3 +86,50 @@ func getGIF(text string) string {
...
+	json.Unmarshal(body, &jsonResponse)
+
+	//var results map[string]interface{} = jsonResponse["results"]
```
Comment (bennyz) [inline main.go]: Please remove

File: main.go [hunk 3]
```diff
@@ -68,3 +86,50 @@ func getGIF(text string) string {
...
+	results := jsonResponse.Results
+
+	key := text
```
Comment (bennyz) [inline main.go]: what does this do

File: main.go [hunk 4]
```diff
@@ -68,3 +86,50 @@ func getGIF(text string) string {
...
+	textLower := strings.ToLower(text)
+	if (textLower == "sad" || textLower == "happy") {
+		return text;
+	}
+
+	return fetchEmotion(text)
+}
+
+func fetchEmotion(text string) string {
+	url := "https://apiv2.indico.io/emotion"
```
Comment (bennyz) [inline main.go]: Perhaps a const?

File: main.go (hunk 3 above)
Comment (matanshukry) [reply main.go]: Declaration + default value if results is empty?

**Pull request merged**